controllers/contactsControllers.js

```diff
@@ -0,0 +1,53 @@
+const { HttpError, ctrlWrapper } = require("../utils");
+const {
+  listContacts,
+  getContactById,
+  addContact,
+  updateContact,
+  removeContact,
+} = require("../models/contacts");
+
+const getList = async (req, res, next) => {
+  const contacts = await listContacts();
+  res.json(contacts);
+};
+
+const getById = async (req, res, next) => {
```

Comment:
```suggestion
const getСontactById = async (req, res, next) => {
```
назви інших контроллерів аналогічно виправте